documentation/homework/week3/todo/src/Params/locals.js

```diff
@@ -0,0 +1,54 @@
+
+class Params {
+  get random_8() { // just for Id's For Now
```

Comment:
This is probably a "safe" way to generate unique IDs, without looking into it too deep... But just so you are aware, there are lots and lots of popular libraries to do things like this, where another developer has spent many hours solving annoying problems so that you don't have to.

I think [uuid](https://www.npmjs.com/package/uuid) is probably the most popular. 1.5 million downloads per day.